pkg/app/server.go

```diff
@@ -371,3 +395,12 @@ func buildMetricsServer(m *metricshandler.MetricsHandler, durationObserver prome
 	})
 	return mux
 }
+
+func md5HashAsMetricValue(data []byte) float64 {
+	sum := md5.Sum(data) //nolint:gosec
+	// We only want 48 bits as a float64 only has a 53 bit mantissa.
+	smallSum := sum[0:6]
```

Comment:
What about `0:7` to include all 53 bits of the mantissa? We'd waste a bit, sure, but this would make sure the hash is more unique.

Reply:
I'm not sure this will matter much practically.